subt/cloudsim2osgar.py

```diff
@@ -87,6 +88,7 @@ def __init__(self, robot_name):
         # common topics
         topics = [
             ('/' + robot_name + '/imu/data', Imu, self.imu, ('rot', 'acc', 'orientation')),
+            ('/' + robot_name + '/odom_fused', Odometry, self.odom_fused, ('pose3d',)),
```

Comment:
Adding this to common topics while only publishing it for X4 breaks all other robots. This needs to go to

```
elif "ssci_x4_sensor_config_2"
```

section that has been created for the sole purpose of supporting messages specific to the X4.

Reply:
I am doing some local testing so I'll fix this.

Reply:
Fixed in f65d0d1